protocol-units/bridge/contracts/src/IAtomicBridgeInitiator.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.22;
+
+interface IAtomicBridgeInitiator {
+    // Event emitted when a new atomic bridge transfer is created
+    event BridgeTransferInitiated(
+        bytes32 indexed _bridgeTransferId,
+        address indexed _originator,
+        bytes32 indexed _recipient,
+        uint256 amount,
+        bytes32 _hashLock,
+        uint256 _timeLock
+    );
+    // Event emitted when a BridgeTransfer is completed (withdrawn)
+    event BridgeTransferCompleted(bytes32 indexed _bridgeTransferId, bytes32 pre_image);
+    // Event emitted when a BridgeTransfer is refunded
+    event BridgeTransferRefunded(bytes32 indexed _bridgeTransferId);
+
+    error ZeroAmount();
+    error WETHTransferFailed();
+    error BridgeTransferInvalid();
+    error InvalidSecret();
+    error BridgeTransferHasBeenCompleted();
+    error TimeLockNotExpired();
+    error ZeroAddress();
+    error Unauthorized();
+
+    /**
+     * @dev Creates a new atomic bridge transfer using native ETH
+     * @param _wethAmount The amount of WETH to send
+     * @param _recipient The address on the other chain to which to transfer the funds
+     * @param _hashLock The hash of the secret (HASH) that will unlock the funds
+     * @param _timeLock The number of blocks until which this BridgeTransfer is valid and can be executed
```

Comment:
This says number of blocks for `_timeLock` but in the code implementation it seems to be using time in seconds. It seems like research is needed to be sure about which approach will work best.

Reply:
Yep, well noticed. On a discussion with @cde8  we've decided to go with `block.timestamp` on eth side and the now timestamp on move which is in UTC secs. It will only be in e2e testing where we see if there are any issues with it. We walked through it yesterday and it seems to be solid.